src/components/viewer/Attachment/index.js

```diff
@@ -143,10 +143,10 @@ export default class Attachment extends React.PureComponent {
         return !viewOnly && this.isTabularCompatible(response.fileType);
     }
 
-    shouldHideTabularButton = ({ isEarlyAccess }) => {
+    shouldHideTabularButton = () => {
         const { tabularBook } = this.props;
         const isAlreadyExtracted = !!tabularBook;
-        return !(isEarlyAccess && (isAlreadyExtracted || this.isExtractable()));
+        return !((isAlreadyExtracted || this.isExtractable()));
```

Comment:
`!((isAlreadyExtracted || this.isExtractable()))` has a redundant bracket enclosing

Reply:
It's better to simplify this to `!isAlreadyExtracted && !this.isExtractable()`